commands-yml/commands/mobile-command.yml

```diff
@@ -78,6 +84,7 @@
       | mobile:viewportScreenshot | Like [screenshot](/commands/session/screenshot/) but only includes contents of viewport | <none> | <none> |
       | mobile:deepLink | Opens a deep-link URL for testing [Instant Apps](https://support.google.com/googleplay/answer/7240211?hl=en) | `{url, package}` | `{url: "https://www.site.com/", package: "com.site.SomeAndroidPackage"}` |
       | mobile:deviceInfo | Gets device information like manufacturer, model, timezone and locale. Read [GetDeviceInfo](https://github.com/appium/appium-uiautomator2-server/blob/master/app/src/main/java/io/appium/uiautomator2/handler/GetDeviceInfo.java) for more details. | <none> | <none> |
+      | mobile:type | Types the given Unicode string into focused field. | `{text}` | `{text: 'happy testing'}` |
```

Comment:
We could also mention the purpose of this endpoint: https://github.com/appium/appium-uiautomator2-driver/pull/365#issuecomment-589926654 (not sure how to describe it briefly though :) )

Reply:
Good idea 💡